src/app/workloads/workloads.component.ts

```diff
@@ -32,20 +32,22 @@ import { ISubscription } from "rxjs/Subscription";
   styleUrls: ["./workloads.component.scss"]
 })
 export class WorkloadsComponent implements OnInit, OnDestroy {
-  private podUnsub: ISubscription;
-  private timeUnsub: ISubscription;
-  private litmustimeUnsub: ISubscription;
-  private litmusUnsub: ISubscription;
+  private podUnsub$: ISubscription;
+  private timeUnsub$: ISubscription;
+  private litmustimeUnsub$: ISubscription;
+  private litmusUnsub$: ISubscription;
   private routeSub: ISubscription;
-  private personalSub: ISubscription;
-  private pvcDetails: ISubscription;
+  private personalSub$: ISubscription;
+  private pvcDetails$: ISubscription;
+  private mayactlVolumeList$: ISubscription;
```

Comment:
I think **$** is not required for private variable, standard practice is use only the camel case
ref: https://v2.angular.io/docs/ts/latest/guide/style-guide.html

Reply:
it's observale

Reply:
In observable also they are not using any **$** sign